### DISCUSSION TITLE
set /etc/grafana/provisioning/ to mode 1777 rather than 777

### DISCUSSION THREAD

**Discussion by fabienhinault** [grafana/grafana#51908] [General]
Hello,
Some security tools (at customers') warn against directories in mode 777 in docker images.
/etc/grafana/provisioning/ and other directories are mode 777 in grafana's docker image.
see https://github.com/grafana/grafana/blob/main/Dockerfile#L85
Setting these directories to 1777 would mute those security tools.
Hopefully, it would not harm any feature.
What do you think?

--Fabien

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
